### PR TITLE
Updates for OpenTofu runner

### PR DESCRIPTION
**Pull request by sudhir-kelkar**

### CODE REVIEW

File: manageiq-operator/api/v1alpha1/helpers/miq-components/orchestrator.go [hunk 1]
```diff
@@ -3,6 +3,9 @@ package miqtools
 import (
 	"context"
 
+	"strconv"
```
Comment (bdunne) [inline manageiq-operator/api/v1alpha1/helpers/miq-components/orchestrator.go:6]: Any reason these moved?

File: manageiq-operator/api/v1alpha1/manageiq_types.go [hunk 2]
```diff
@@ -331,6 +331,11 @@ type ManageIQSpec struct {
 	// +optional
 	UIWorkerImage string `json:"uiWorkerImage,omitempty"`
 
+	// Image string used for the Opentofu runner worker deployments
```
Comment (bdunne) [inline manageiq-operator/api/v1alpha1/manageiq_types.go:334]: Please sort alphabetically

File: manageiq-operator/config/crd/bases/manageiq.org_manageiqs.yaml [hunk 3]
```diff
@@ -314,6 +314,11 @@ spec:
                   Image string used for the UI worker deployments
                   By default this is determined by the orchestrator pod
                 type: string
+              opentofuRunnerImage:
```
Comment (bdunne) [inline manageiq-operator/config/crd/bases/manageiq.org_manageiqs.yaml:317]: Tests are failing because this is not sorted alphabetically

Comment (bdunne) [reply manageiq-operator/config/crd/bases/manageiq.org_manageiqs.yaml:317]: @Fryguy Any thoughts on the naming?  Not sure if we prefer `opentofu`, `terraform` or something else.

**Review by bdunne — COMMENTED** [review]
Is it just the test failure or is there another reason this is marked as WIP?

**Comment by sudhir-kelkar** [comment]
> Is it just the test failure or is there another reason this is marked as WIP?

For Test Failure... My Dev environment crashed. Creating new environment which will have docker on it.

File: manageiq-operator/api/v1alpha1/helpers/miq-components/orchestrator.go (hunk 1 above)
Comment (sudhir-kelkar) [reply manageiq-operator/api/v1alpha1/helpers/miq-components/orchestrator.go:6]: Line swapping is happening during file save so kept it like that.

File: manageiq-operator/config/crd/bases/manageiq.org_manageiqs.yaml (hunk 3 above)
Comment (sudhir-kelkar) [reply manageiq-operator/config/crd/bases/manageiq.org_manageiqs.yaml]: Thanks... Tests are passing now

Comment (Fryguy) [reply manageiq-operator/config/crd/bases/manageiq.org_manageiqs.yaml]: I think opentofuRunnerImage is good and it matches the class name on the other side...however would also like @agrare's thoughts

**Comment by Fryguy** [comment]
@sudhir-kelkar Please do a rebase instead of merging master into the branch as per https://github.com/ManageIQ/manageiq-pods/pull/1067#issuecomment-1994821363

File: manageiq-operator/config/crd/bases/manageiq.org_manageiqs.yaml (hunk 3 above)
Comment (agrare) [reply manageiq-operator/config/crd/bases/manageiq.org_manageiqs.yaml]: Yeah I named the class `OpentofuRunner` on the ruby side so `opentofuRunnerImage` makes sense to me

File: manageiq-operator/api/v1alpha1/helpers/miq-components/orchestrator.go (hunk 1 above)
Comment (agrare) [reply manageiq-operator/api/v1alpha1/helpers/miq-components/orchestrator.go:6]: That doesn't sound like a good reason to move it and more of a local editor issue on your side.

**Comment by Fryguy** [comment]
Backported to `radjabov` in commit 0667319181739d66d9f40df9f9563fe9c0b73204.

```text
commit 0667319181739d66d9f40df9f9563fe9c0b73204
Author: Brandon Dunne <brandondunne@hotmail.com>
Date:   Mon Mar 18 14:04:10 2024 -0400

    Merge pull request #1067 from sudhir-kelkar/opentofu-runner
    
    Updates for OpenTofu runner
    
    (cherry picked from commit f999473017ea779efda8cfb37619cce551d19a93)
```

**Pull request merged**